amazon-ecr-credential-helper.spec

```diff
@@ -139,6 +139,9 @@ install -D -m 0644 \
 rm -rf %{buildroot}
 
 %changelog
+* Tue May 21 2024 Ke Xu <kkexu@amazon.com> - 0.7.1-4
+- Rebuild with golang-1.22.3
+- Fix CVE-2023-45288
```

Comment:
Changelog is mainly for the core package features/bug-fixes/CVE fixes. We do not update changelog for build version update.

Reply:
Removed the changelog entry